src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -1481,17 +1480,18 @@ featurizers:
           - float
           - double
 
+    # TODO: Output type should be changed to matrix<nonstd::optional<T>> and input should support all types after yaml file support nonstd::optional type

```

Comment:
Isn't this always going to return a type double and not T? I thought that they said that this doesn't make sense for non-numeric values? Or are we just going to change to T so that we can support it even if its not something they use?

Reply:
Given that it is just copying values, it seems reasonable that it could copy any type and not just doubles.